pkg/services/sqlstore/migrations/accesscontrol/dashboard_permissions.go

```diff
@@ -57,6 +55,7 @@ var folderPermissionTranslation = map[models.PermissionType][]string{
 func AddDashboardPermissionsMigrator(mg *migrator.Migrator) {
 	mg.AddMigration("dashboard permissions", &dashboardPermissionsMigrator{})
 	mg.AddMigration("dashboard permissions uid scopes", &dashboardUidPermissionMigrator{})
+	mg.AddMigration("drop managed folder create actions", &managedFolderCreateAction{})
```

Comment:
nit: again, I don't think it matters in this case, but we should aim to add new migrations to the end of the full migration list to ensure the migration order.

Reply:
It does not matter in this case, we just need to make sure we runt this after the other two